src/client/Microsoft.Identity.Client/Internal/Requests/RequestBase.cs

```diff
@@ -355,7 +355,8 @@ private void LogRequestStarted(AuthenticationRequestParameters authenticationReq
                 !authenticationRequestParameters.IsClientCredentialRequest &&
                 !CacheManager.TokenCacheInternal.IsAppSubscribedToSerializationEvents())
             {
-                authenticationRequestParameters.RequestContext.Logger.Error("The default token cache provided by MSAL is not designed to be performant when used in confidential client applications. Please use token cache serialization. See https://aka.ms/msal-net-cca-token-cache-serialization.");
+                authenticationRequestParameters.RequestContext.Logger.Warning(
+                    "Only in-memory caching is used. The cache is not persisted and will be lost if the machine is restarted. It also does not scale for a web app or web api, where the number of users can grow large. In production, web apps and web apis should use distributed caching like Redis. See https://aka.ms/msal-net-cca-token-cache-serialization");
```

Comment:
nit:
```suggestion
                    "Only in-memory caching is used. The cache is not persisted and will be lost if the machine is restarted. It also does not scale for a web app or web API, where the number of users can grow large. In production, web apps and web APIs should use distributed caching like Redis. See https://aka.ms/msal-net-cca-token-cache-serialization");
```

Reply:
Thanks @pmaytak . I will take the changes, but not via suggestion. I think GH messes up the spacing when you take suggestions. See @trwalke 's PR on the CacheSyncronization option.